group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -177,11 +238,22 @@ GroupMetadataManager build() {
                 consumerGroupMaxSize,
                 consumerGroupSessionTimeoutMs,
                 consumerGroupHeartbeatIntervalMs,
-                consumerGroupMetadataRefreshIntervalMs
+                consumerGroupMetadataRefreshIntervalMs,
+                topicPartition,
```

Comment:
extremely small nit: Should you move `topicPartition` to the top? This is a common attribute.